content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -23,3 +23,122 @@ All students need to provably understand all of the following concepts:
 - getters and setters
 - inheritance and overriding and extending methods
 - composition
+
+# Python
+
+## Constructors
+The `__init__()` method is called the constructor and is always called when an object is created, also known as instantiating an object.
```

Comment:
This is mostly true... there is actually a __new__ function that's the real constructor. __init__is the initialiser.

In python we dont use the constructor very often. Only if you are doing something unusual.  So we should focus on __init__ and ignore __new__. But just be sure to use the corect terminology